### ISSUE TITLE
abduco/dvtm

### ISSUE THREAD

**Issue by jgarte** [maciejjan/vis-tmux-repl#1]
Hi, this a great idea. Thank you for making this!

Do you think it would be possible to implement this with `abduco` and `dvtm` instead?

In other words, is there something missing from abduco and/or dvtm that tmux has in oder to send to a repl?

**Comment by maciejjan** [comment]
Yes. Tmux has a scripting interface, so you can call its commands from outside of the tmux process by executing the tmux binary. For example you can call `tmux split` from whereever and it will split the currently focused pane in your active tmux session.

This plugin relies in particular on the command `tmux send-keys` that is executed from vis to simulate keypresses to another terminal (the one where the REPL is running).

I'm not very familiar with dvtm, but I don't think it has these functionalities.

Thank you for your interest!
